processor/parse/flavored/callout.js

```diff
@@ -1,5 +1,30 @@
+// @note: We'd like to allow any emoji to match, but included in the emoji
+// character set is [#*0-9].
+// https://www.unicode.org/Public/13.0.0/ucd/emoji/emoji-data.txt
 // eslint-disable-next-line unicorn/no-unsafe-regex
-const rgx = /^> ?((?:\u2139|\u2049|\u2757|\u203C|\u00a9|\u00ae|[\u2000-\u3300]|\ud83c[\ud000-\udfff]|\ud83d[\ud000-\udfff]|\ud83e[\ud000-\udfff])+)(?: {0,}(.+))?\n((?:>(?: .*)?\n)*)/;
+const rgx = /^> ?((?![#*0-9])\p{Emoji})(?: {0,}(.+))?\n((?:>(?: .*)?\n)*)/u;
```

Comment:
this is slightly better than it used to be! bummer `{Emoji}` wasn't the magic solution we hoped it would be.